doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc

```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
 ====
 
 {KIE_SERVER} connects to the new {CENTRAL} and collects the list of deployment units (KIE containers) to be deployed.
+
+ifdef::PAM[]
+[NOTE]
+====
+When the class is used inside a dependency JAR file to access {KIE_SERVER} from {KIE_SERVER} client, you get the `ConversionException` and `ForbiddenClassException` in {CENTRAL}.
```

Comment:
When you use a class inside....

Use active voice.

Reply:
Done